README.md

```diff
@@ -1,5 +1,51 @@
-# Mini-Projects
+# Machine Learning Engineering Course Mini-Projects
+
+This is the repository of the current miniprojects for Springboard's Machine Learning Engineering course. As you work through the curriculum, you will need to complete each and everyone of those mini-projects and review and discuss them with your mentor.
```

Comment:
@arnaudsj can you remove Springboard name from this? and just say "This is the repository of the current miniprojects for the Machine Learning Engineering course."

Reply:
Got it!